packages/squiggle-lang/src/public/SqProject/index.ts

```diff
@@ -276,83 +289,112 @@ export class SqProject {
     return Result.fmap(this.getOutput(sourceId), ({ bindings }) => bindings);
   }
 
-  private async buildExternals(
-    sourceId: string,
+  getImportBindings(sourceId: string): Result.result<SqDict, SqError> {
```

Comment:
This function is:
- not used anywhere and not tested
- identical to `getBindings`

Reply:
Yep, was an accident, thanks!